sdk-lib/src/main/java/cash/z/ecc/android/sdk/Synchronizer.kt

```diff
@@ -838,14 +839,18 @@ interface Synchronizer {
 
             val encoder = DefaultSynchronizerFactory.defaultEncoder(backend, saplingParamTool, repository)
 
-            val txManager = DefaultSynchronizerFactory.defaultTxManager(encoder, walletClient)
+            val sdkFlags = SdkFlags(
+                isTorEnabled = isTorEnabled
```

Comment:
Same review comment applies here as I made on the Swift SDK (https://github.com/Electric-Coin-Company/zcash-swift-wallet-sdk/pull/1578#discussion_r2207649794): `TorClient.new` is always called regardless of this flag, and thus is always running (and observable as such by network adversaries) whether or not we make use of it.